backend/model/model.go

```diff
@@ -886,8 +886,9 @@ type ErrorObject struct {
 	SessionID        *int
 	TraceID          *string
 	SpanID           *string
-	LogCursor        *string `gorm:"index:idx_error_object_log_cursor,option:CONCURRENTLY"`
-	ErrorGroupID     int     `gorm:"index:idx_error_group_id_id,priority:1,option:CONCURRENTLY"`
+	LogCursor        *string    `gorm:"index:idx_error_object_log_cursor,option:CONCURRENTLY"`
+	ErrorGroupID     int        `gorm:"index:idx_error_group_id_id,priority:1,option:CONCURRENTLY"`
+	ErrorGroup       ErrorGroup `gorm:"foreignkey:ErrorGroupID"`
```

Comment:
Defining FK so we can preload the error group later on.

Reply:
is there any db blocking while the foreign key constraint is added or chance it will fail (e.g. if any error objects reference a null or a non-existent error group id)? I think it would be good to add this FK manually before deploying this change